podium-gateway/src/main/webapp/app/request/main-detail/detail/request-detail.component.html

```diff
@@ -93,14 +93,14 @@ <h3 jhiTranslate="request.relatedRequests">Related requests</h3>
                             <td>
                                 <ul>
                                     <li *ngFor="let requestType of relatedRequest.requestDetail.requestType">
-                                        {{ requestType | capitalize }}
+                                        {{ requestType | titlecase }}
```

Comment:
The types are also already capitalised, no need to do it here.
```suggestion
                                        {{ requestType }}
```